packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/LightCurveViewer/LightCurveViewerContainer.js

```diff
@@ -94,42 +128,82 @@ class LightCurveViewerContainer extends Component {
   }
 
   render () {
-    const { subject } = this.props
+    const {
+      addAnnotation,
+      annotations,
+      currentTask,
+      drawFeedbackBrushes,
+      enableAnnotate,
+      enableMove,
+      feedback,
+      interactionMode,
+      onKeyDown,
+      setOnPan,
+      setOnZoom,
+      subject,
+      toolIndex,
+    } = this.props
```

Comment:
The LCV has a scary number of props. That runs through my mind whenever I work on it.

Reply:
Yep, next steps will be refactor on some of this. Inevitable for this step though.